libs/network/esp32/jswrap_esp32_network.c

```diff
@@ -508,6 +512,8 @@ static esp_err_t event_handler(void *ctx, system_event_t *event)
     jsvObjectSetChildAndUnLock(jsDetails, "mac", jsvNewFromString(temp));
     sprintf(temp, "%d", event->event_info.disconnected.reason);
     jsvObjectSetChildAndUnLock(jsDetails, "reason", jsvNewFromString(temp));
+    sprintf(temp,"%s",wifiReasonToString(event->event_info.disconnected.reason));
```

Comment:
I think `wifiReasonToString` now returns strings that are over 35 chars in length with the new error messages? That means it'll overflow `temp` because `sprintf` doesn't check bounds - `temp` needs to be bigger

Reply:
Actually scratch that - you can just do `jsvObjectSetChildAndUnLock(jsDetails, "msg", jsvNewFromString(wifiReasonToString(event->event_info.disconnected.reason))))` ?